apps/web-portal-client/package.json

```diff
@@ -16,22 +16,25 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@reduxjs/toolkit": "^1.5.0",
     "classnames": "^2.2.6",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-redux": "^7.2.2",
     "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0"
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.5"
   },
   "devDependencies": {
-    "@zorko-io/util-web-api-client": "1.0.0",
-    "@zorko-io/util-validation": "1.0.0",
     "@babel/core": "^7.11.1",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-syntax-class-properties": "^7.12.1",
     "@babel/polyfill": "^7.12.1",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
+    "@zorko-io/util-validation": "1.0.0",
```

Comment:
looks like they should go to the ordinary dependencies section

Reply:
done